packages/@okta/courage/package.json

```diff
@@ -1,56 +1,61 @@
 {
-  "_args": [
-    [
-      "@okta/courage@3.3.0-beta.12",
-      "/home/vagrant/okta/okta-signin-widget"
-    ]
-  ],
-  "_from": "@okta/courage@3.3.0-beta.12",
-  "_id": "@okta/courage@3.3.0-beta.12",
-  "_inCache": true,
-  "_installable": true,
+  "_from": "@okta/courage@4.1.0-beta.1",
+  "_id": "@okta/courage@4.1.0-beta.1",
+  "_inBundle": false,
+  "_integrity": "sha1-DZJWjN88Klai6jAghhPcperTHRs=",
   "_location": "/@okta/courage",
   "_phantomChildren": {
     "amdefine": "1.0.1",
+    "read-installed": "4.0.3",
+    "request": "2.67.0",
+    "underscore": "1.8.3",
+    "walkdir": "0.0.7",
     "wordwrap": "0.0.3"
   },
   "_requested": {
+    "type": "version",
+    "registry": true,
+    "raw": "@okta/courage@4.1.0-beta.1",
     "name": "@okta/courage",
-    "raw": "@okta/courage@3.3.0-beta.12",
-    "rawSpec": "3.3.0-beta.12",
+    "escapedName": "@okta%2fcourage",
     "scope": "@okta",
-    "spec": "3.3.0-beta.12",
-    "type": "version"
+    "rawSpec": "4.1.0-beta.1",
+    "saveSpec": null,
+    "fetchSpec": "4.1.0-beta.1"
   },
   "_requiredBy": [
+    "#USER",
```

Comment:
I still haven't look into if this value matters or not since we are using ```packages``` folder, but if it does it should be kept as ```/``` (long explanation: ```/``` means the i18n module is required by the root module. ```#USER``` means we ran 'npm install' but we did not save the installed module)